### DISCUSSION TITLE
J5: Only Introtext is shown on the item when I set the introtext in the categorylist to a special value

### DISCUSSION THREAD

**Discussion by litho-publisher** [eworkers/K2ForJ4#67] [General]
Hello
and many thanks for your work! I have tried to go back to Joomla content, but it is too cumbersome for my clients. So I tested K2forJ4 and updated to J5. It worked very well. There was only one problem.
If I display the intro text in the category lists with e.g. 20 words, the main article also only shows 20 words and not the full text. Compared to the older K2 version, I could not see any error in item.php, I thought that maybe the intro text was confused with full text, but that was not the case. 

I will test it again with a Joomla 4 version. But maybe someone has an idea about my J5 problem. 
Thanks a lot
regards
litho

p.s. how can I support you? is there a donation account?

**Comment by litho-publisher** [comment]
ok I have tested it with the latest Joomla 4.4.3, php 8.1 version.
As soon as I enter a value in the category -> Display stettings -> Item view options in category listings -> Introtext word limit, the intro text is displayed with the corresponding value, but also with the same value when I display the item. The full text is not displayed in the article.
If I enter a manual "read more", the display in the category list as well as in the article is ok. The automatic separation of the introductory text after X words also leads to separation in the article and the full article text is not displayed.

Ops, could it be that it will works with php 8.3? I even tested it and it seems to work.

**Comment by eworkers** [comment]
please update and check if e652889 solves your issue

**Comment by litho-publisher** [comment]
yes it solved the problem, tested with php 8.1,2,3 and J4 anf J5
Thanks!!
